examples/stable_diffusion_v2/ldm/modules/train/callback.py

```diff
@@ -161,12 +185,11 @@ def on_train_step_end(self, run_context):
 
                 self.step_start_time = time.time()
                 _logger.info(
-                    "epoch: %d step: %d, loss is %.3f, average step time (in %d step(s)): %.3f.",
+                    "epoch: %d step: %d, loss is %.6f",
```

Comment:
why delete `average step time` ?

Reply:
It's to make the log cleaner. When dataset sink is ON, it's not used. When OFF, the average step time still flunctuates with default log_interval. So it's better to get the mean value from the recorded result.log